pages/docs/installation/installation-source-configuration.md

```diff
@@ -7,6 +7,26 @@ keywords: configuration, basics, cmake, installation, building, source
 preCICE uses [CMake](https://cmake.org/) to configure and build the library.
 After this step, preCICE is ready to [be built](installation-source-building).
 
+## Using presets
+
+preCICE offers a range of CMake configuration presets, which all use the build directory `build`.
```

Comment:
Maybe add a statement on what these presets actually set.

Reply:
Do they affect the prefix? That would be useful at this stage of the documentation.